scripts/bootstrap/bootstrap

```diff
@@ -35,7 +35,7 @@ get-branch() {
     local branch
     if [[ -f "$opkg_conf_path" ]]; then
         # Get the first (...) value (aka \1) of the pattern if matching
-        branch="$(sed -E 's|^src/gz\s+toltec\s+https://toltec-dev.org/([[:alnum:]]*).*$|\1|;t;d' "$opkg_conf_path" | head -n1)"
```

Comment:
Perhaps get-branch should allow checking for specific items (i.e. `rm1`, `rm2`, and `rmall`)? And where it's used should be a little smarter about which one it's checking and replacing?

Reply:
It’s a fair point, but installs using different branches for rm1, rm2 and rmall should be fairly rare, if they exist at all. I think the current behavior is fine for now, we’ll be able to implement more complex behaviors when we move this logic to a specific script for handling branches.